schema/artist/index.js

```diff
@@ -111,6 +111,23 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      formatted_nationality_and_birthday: {
+        type: GraphQLString,
+        resolve: ({ birthday, nationality }) => {
+          let formatted_bday = (!isNaN(birthday) && birthday) ? 'b. ' + birthday : birthday;
+
+          if (formatted_bday && formatted_bday.toLowerCase().includes('born')) {
```

Comment:
hmm I played around with it. If you also go with @alloy's suggestion below, where you pass it the regex inline, it means that you won't need to check or differentiate between `born` and `Born`, in which case, `replace()` will either do it or leave the string alone :). 

```
 if (formatted_bday) {
   formatted_bday = formatted_bday.replace(/born/i, 'b.');
}
```

I'm not 100% about having to check whether it isn't null, but I assume you wouldn't be able to call `replace()` on it otherwise?